src/components/organisms/AssetActions/Consume.tsx

```diff
@@ -11,6 +11,24 @@ import { useOcean, useConsume, usePricing } from '@oceanprotocol/react'
 import { useSiteMetadata } from '../../../hooks/useSiteMetadata'
 import checkPreviousOrder from '../../../utils/checkPreviousOrder'
 import { useAsset } from '../../../providers/Asset'
+import { mapSecondsToTimeoutString } from '../../../utils/metadata'
+
+function getHelpText(
```

Comment:
ok, now we are here. Revised that whole logic we are using to display help text and introduced another sentence which considers the timeout too. Totally needs testing but think then it all makes sense with all the combinations of button + help text

Reply:
went through all 3 scenarios and it seems ok